src/msg/node.h

```diff
@@ -25,9 +25,13 @@ class CRNode {
     using job_t           = JobRunner::job_t;
     using strand_ptr_t    = std::shared_ptr<JobRunnerStrand>;
 
-    explicit CRNode() : CRNode("noname") {}
+    explicit CRNode() : CRNode("noname", nullptr) {}
```

Comment:
Why explicit a def ctor?

Reply:
Unlike the single parameter ctor, I don't have a strong opinion here, but:

- explicit to disallow conversion from `{}`
- just in case the parameter list changes. Or we may define another single-parameter ctor by copying and modifying this one. We may forget to add `explicit`

Reply:
In this case, it is probably copied from another explicit ctor, and I don't feel I need to remove the specifier.